src/views/workflow-history-v2/workflow-history-event-group/workflow-history-event-group.types.ts

```diff
@@ -4,14 +4,19 @@ import { type HistoryEventsGroup } from '@/views/workflow-history/workflow-histo
 import { type Props as WorkflowHistoryProps } from '../workflow-history-v2.types';
 
 export type Props = {
+  // Core data props
```

Comment:
Just rearranged & grouped the props here 😅

Reply:
Thx for context :)